src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -6,6 +6,13 @@ if (NOT CLR_CMAKE_USE_SYSTEM_ZLIB)
     include(${CLR_SRC_NATIVE_DIR}/external/zlib-ng.cmake)
 endif()
 
+if (CLR_CMAKE_USE_SYSTEM_BROTLI)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(BROTLI REQUIRED brotlidec brotlienc brotlicommon)
```

Comment:
I was looking for these two!

Why here and not in extra_libs like with zlib-ng?

Reply:
I was having weird issues with it being in extra_libs, and using `PUBLIC` in the `target_link_libraries` and `target_include_directories` below got the values to propagate such that I didn't need to include `brotli.cmake` in any other place than here.